cypress/e2e/specialFeatures/liteSiteWeight/assertions/liteSiteWeight.js

```diff
@@ -0,0 +1,62 @@
+import interceptGetRequests from '../helpers/interceptGetRequests';
+import getTotalPageSize from '../helpers/getTotalPageSize';
+
+const MAX_PAGE_WEIGHT_KB = 100;
+
+const roundTo2Decimals = num => {
+  return parseFloat(num.toFixed(2));
+};
+
+export default ({ path, pageType }) => {
+  describe('', () => {
+    let allRequests = [];
+    let liveRequests = [];
+    before(() => {
+      Cypress.automation('remote:debugger:protocol', {
+        command: 'Network.setCacheDisabled',
+        params: { cacheDisabled: true },
+      });
+      cy.clearCookies();
+      cy.clearLocalStorage();
+      interceptGetRequests(allRequests);
+      cy.visit(`${path}`);
+    });
+
+    afterEach(() => {
+      allRequests = [];
+      liveRequests = [];
+    });
+
+    it(`Page weight for ${pageType} page should be less than ${MAX_PAGE_WEIGHT_KB}Kb`, () => {
+      let localPageWeight;
+
+      getTotalPageSize(allRequests)
+        .then(localSize => {
+          localPageWeight = roundTo2Decimals(localSize);
+        })
```

Comment:
Can we include the rounding to 2dp in the getTotalSize because we repeat it here and below.